app/controllers/application_controller.rb

```diff
@@ -1,3 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  before_filter :set_customer_api_token
```

Comment:
`before_filter` is deprecated and will be removed in Rails 5.1, better to use `before_action`.